.env.example

```diff
@@ -13,4 +13,30 @@ SERVER_PORT=8080
 DIGITALOCEAN_TOKEN=your_digitalocean_token_here
 
 #Logrus
-LOG_LEVEL=info
\ No newline at end of file
+LOG_LEVEL=info
+
+# Kong DB
+POSTGRES_USER=kong
```

Comment:
For these and others I'd make them more applicable to kong than the underlying service. 

Ie kong_user not postgres_user since postgres could change in the future and on its own does not help us understand the difference between this user and the other db_user env var